dev_guide/templates.adoc

```diff
@@ -97,15 +150,15 @@ by piping both commands:
 $ oc process -f <filename.json> | oc create -f -
 ----
 
-You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option followed by a
+You can override any link:../dev_guide/templates.html#parameters[parameter]
+values defined in the JSON file by adding the `-v` option followed by a
```

Comment:
Good idea re: a "Supported Formats" section.

Re: 3) and 4), I think we're going to try to move to YAML for usage in docs (not sure how soon, have already started using YAML in newer spots), since it's easier to read. But that's a good point in 1) and 2) about what the actual example applications are using...